src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -69,7 +69,7 @@ public function copy($originFile, $targetFile, $overwriteNewerFiles = false)
             }
 
             // Like `cp`, preserve executable permission bits
-            @chmod($targetFile, fileperms($targetFile) | (fileperms($originFile) & 0111));
+            @chmod($targetFile, fileperms($targetFile) | (@fileperms($originFile) & 0111));
```

Comment:
shouldn't the first "fileperms" also be silenced?
would that work?
`@(chmod($targetFile, fileperms($targetFile) | (fileperms($originFile) & 0111))));`